completer.go

```diff
@@ -50,7 +50,7 @@ func (ic iCompleter) getWords(w []string) (s []string) {
 		cmd, args = ic.cmd, w
 	}
 	if cmd.Completer != nil {
-		return cmd.Completer(args)
+		s = append(s, cmd.Completer(args)...)
```

Comment:
I use the completer argument to hide some commands of a command. If we are doing this we should add a hide operations aswell I guess.